cmake/TaichiCore.cmake

```diff
@@ -53,6 +54,21 @@ if (TI_WITH_CUDA)
     endif()
 endif()
 
+if (TI_WITH_OPENGL)
+  if(NOT GLEW_VERSION)
+    set(GLEW_VERSION 2.0.0)
+  endif()
+  find_package(GLEW ${GLEW_VERSION})
+  if (GLEW_FOUND)
+    message("Building with GLEW ${GLEW_VERSION}")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DTI_WITH_OPENGL")
+    target_link_libraries(${LIBRARY_NAME} /usr/lib/libGLEW.so GLEW)
```

Comment:
- Maybe we should make use of variables such as `${GLEW_LIBRARIES}` to avoid hard-coded library path?
- Is it possible to do static linking against `libGLEW.a`? That will make shipping `taichi` much easier.

Reply:
Unlike `libcudart_static.a`, I can't find any file named `libGLEWxxx.a` in my `/lib` dir.

Reply:
For GLEW, simply put `#define GLEW_STATIC` before include will work, acording to https://blog.csdn.net/pc1377318286/article/details/50885395.